src/scripts/slate/cart.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Cart Template Script
+ * ------------------------------------------------------------------------------
+ * A file that contains scripts highly couple code to the Cart template.
+ *
+ * @namespace cart
+ */
+
+slate.cart = {
+  
+  /**
+   * Check if cookies are enabled in the browser
```

Comment:
Please put a small blurb in here that explains browser cookies are required to build a cart